### ISSUE TITLE
Can't find any way to set version

### ISSUE THREAD

**Issue by brianary** [dotnet/cli#12910]
## Steps to reproduce

1. Add `<VersionPrefix>1.0.0</VersionPrefix>` to **netcoreapp3.0** fsproj file
2. `dotnet build Project.fsproj --version-suffix 1` (as [documented](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build "--version-suffix <VERSION_SUFFIX>"))
3. `Get-ChildItem -Recurse -Filter Project.dll |Foreach-Object VersionInfo`

### or

1. Add `<Version>1.0.0</Version>`
2. `dotnet build Project.fsproj`
3. `Get-ChildItem -Recurse -Filter Project.dll |Foreach-Object VersionInfo`

### or

1. `dotnet build Project.fsproj /p:Version=1.0.0`
2. `Get-ChildItem -Recurse -Filter Project.dll |Foreach-Object VersionInfo`

## Expected  behavior

```text
ProductVersion   FileVersion      FileName
--------------   -----------      --------
1.0.0            1.0.0            ...\bin\Debug\netcoreapp3.0\Project.dll
1.0.0            1.0.0            ...\obj\Debug\netcoreapp3.0\Project.dll
```

## Actual behavior

```text
ProductVersion   FileVersion      FileName
--------------   -----------      --------
                                  ...\bin\Debug\netcoreapp3.0\Project.dll
                                  ...\obj\Debug\netcoreapp3.0\Project.dll
```

## Environment data
`dotnet --info` output:

```text
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100
 Commit:    04339c3a26

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100\

Host (useful for support):
  Version: 3.0.0
  Commit:  95a0a61858

.NET Core SDKs installed:
  1.0.0-preview2-003131 [C:\Program Files\dotnet\sdk]
  3.0.100 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 1.0.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by brianary** [comment]
If it helps, the specific repo I'm having this problem in is [brianary/SelectXmlExtensions](https://github.com/brianary/SelectXmlExtensions).

**Comment by livarcocc** [comment]
This issue has been fixed in 3.1.100.

**Comment by brianary** [comment]
Is there a workaround for this that I can use until next May?
